presto-docs/src/main/sphinx/release/release-326.rst

```diff
@@ -0,0 +1,48 @@
+=========================
+Release 326 (25 Nov 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix performance regression in joins. (:issue:`2047`)
```

Comment:
do we want to use ``JOIN`` ?

Reply:
I'm rephrasing as: `Fix performance regression in queries involving ``JOIN``.`